lcd.lua

```diff
@@ -34,23 +34,29 @@ local create_lines = function(text)
 	local line = ""
 	local line_num = 1
 	local tab = {}
-	for word in string.gmatch(text, "%S+") do
-		if string.len(line)+string.len(word) < LINE_LENGTH and word ~= "|" then
-			if line ~= "" then
-				line = line.." "..word
-			else
-				line = word
-			end
-		else
+	if #text <= LINE_LENGTH then
+		for line in string.gmatch(text, " | ") do
```

Comment:
`gmatch` returns each match. It’s not a split function. So this will just return some strings with ` | ` in them, and drop all the text between the separators.

Reply:
Yeah, true, that makes sense :laughing: I think I'm either gonna take this out later or try whether I can see an easy way to change the rest of the word break thing to make it consistent.